src/core/sys/posix/sys/ioctl.d

```diff
@@ -339,6 +333,11 @@ else version (OSX)
         ushort ts_yyy;
     }
 
+    enum uint TIOCGSIZE = TIOCGWINSZ;
+    enum uint TIOCSSIZE = TIOCSWINSZ;
+
+    import core.sys.posix.sys.filio; // File related ioctls
```

Comment:
On Thu, Jan 28, 2016 at 08:44:12PM -0800, sanjayss wrote:
> I am new to D, so maybe I am missing something related to modules that you are pointing out -- would it be problematic for people to include both, say, filio.d and ioctl.d if they want access to the file related ioctls?

No, it is ok to import both, just in C any #includes inside an included file are available without needing to be included again, whereas on D, you only get that behavior on public imports... so just want to make sure you're aware of that and you are doing what you want it to do.